src/graceful_termination.jl

```diff
@@ -61,6 +64,34 @@ container hook. Typically, K8s initiates graceful termination via the `TERM` sig
 as Julia forcefully terminates when receiving this signal and Julia does not support
 user-defined signal handlers we utilize `preStop` instead.
 
+### Using superviser entrypoint
+
+K8sDeputy provides a bash script in bin/superviser.sh which handles the `TERM` signal and
+sends the `"terminate"` message to the graceful termination socket.  This can be used as the
+`ENTRYPOINT` for your docker image.  For example, after installing `K8sDeputy` in the active
+Julia project:
+
+```dockerfile
+RUN julia --color=yes -e 'using K8sDeputy; K8sDeputy.install_supervise_shim("/usr/bin")'
+ENTRYPOINT ["/usr/bin/supervise.sh"]
+```
+
+The command/script to run and any arguments it requires should be passed in via `args` in
+your [K8s Container
+spec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.33/#container-v1-core);
+anything you specify for `command` in the Container spec will _override_ the container's
+entrypoint.
+
+!!! note
+    The entrypoint script requires `netcat`, which is not present in e.g. `debian-slim`
+    images.  In particular, it requires the "OpenBSD" flavor (available as `netcat-openbsd`
+    in `apt`).  The script will fail before starting if `command -v nc` fails.
```

Comment:
I just ran into an issue using `apt` in a Dockerfile due to the CLI not being stable for scripts. Better to reference `apt-get` to avoid causing our users problems:
```suggestion
    via `apt-get`).  The script will fail before starting if `command -v nc` fails.
```